ArmoryQt.py

```diff
@@ -216,7 +216,8 @@ def __init__(self, parent=None, splashScreen=None):
       self.lockboxIDMap = {}
       self.cppLockboxWltMap = {}
       self.broadcasting = {}
-      self.wallets = WalletMap(self)
+      self.wallets = wallets if wallets is not None else WalletMap()
```

Comment:
No. Raise if the wallets object isn't set! This isnt negotiable, stop trying accommodate for impossible context.